_includes/header--help.html

```diff
@@ -67,7 +67,7 @@
 
       <a
         class="usa-link desktop:display-none padding-y-1 margin-y-3 text-no-underline text-bold"
-        href="{{ '/contact/' | locale_url }}"
+        href="{{ '/partners/contact/' | locale_url }}"
```

Comment:
Based on the ticket, my understanding is we'd only want to update links on the partner subsite, not the base site (only `/partners/` pages).

```suggestion
        href="{{ '/contact/' | locale_url }}"
```

Reply:
Ah you're right... I see now. updated.